YesterdayApi/Utilities/Exceptions/CustomErrorResponse.cs

```diff
@@ -0,0 +1,14 @@
+namespace YesterdayApi.Utilities.Exceptions
+{
+    public class CustomErrorResponse
+    {
+        public string Message { get; set; }
+        public string Description { get; set; }
+        public override bool Equals(object obj)
```

Comment:
Why are you overwriting this? Use generators for that:
<img width="637" alt="screenshot 2018-12-15 at 14 04 17" src="https://user-images.githubusercontent.com/14861637/50043331-5131ca00-0072-11e9-89d3-61dced9065bb.png">

<img width="910" alt="screenshot 2018-12-15 at 14 02 48" src="https://user-images.githubusercontent.com/14861637/50043320-1def3b00-0072-11e9-9452-db76dffe10e5.png">